ext/opcache/zend_accelerator_module.c

```diff
@@ -182,7 +182,7 @@ static ZEND_INI_MH(OnUpdateCounter)
 		*p = val;
 		return SUCCESS;
 	}
-	zend_error(E_WARNING, "Invalid \"%s\" setting. Should be between 0 and 256", ZSTR_VAL(entry->name));
+	zend_error(E_WARNING, "Invalid \"%s\" setting; use default value instead. Should be between 0 and 255", ZSTR_VAL(entry->name));
```

Comment:
I'm not sure, but maybe
```suggestion
	zend_error(E_WARNING, "Invalid \"%s\" setting; using default value instead. Should be between 0 and 255", ZSTR_VAL(entry->name));
```
is more clear about what's happening. /cc @kocsismate

Reply:
Yeah, `using` is better. I am not sure `invalid` is necessary anymore, so here's my preference:
```
zend_error(E_WARNING, "\"%s\" setting should be between 0 and 255. Using default value instead.", ZSTR_VAL(entry->name));
```
This is simple and consistent with the wording we use lately. Feel free to go with Christoph's suggestion though.

Reply:
@kocsismate  Thanks for the review and suggestion; 
zend_error will append some text to your sentence template and finally print out a string like:
_opcache.jit_hot_func setting should be between 0 and 255. Using default value instead **in php.ini on line xx**_

So, I personally prefer Christoph's suggestion, but Christoph's expertise will have the final words.